domain/src/Services/TransactionDispatcher/Handlers/Exceptions/IncomeHandlerException.php

```diff
@@ -14,11 +14,6 @@ private function __construct(string $message)
         parent::__construct($message);
     }
 
-    public static function operationIsNotReceive(Transaction $transaction): self
-    {
-        return new self(sprintf('The transaction is not a receive operation: %s', $transaction->__toString()));
-    }
```

Comment:
ℹ️ Moving to more specific value objects made some checks redundant, like this one